buildtest/schemas/examples/script.schema.json/invalid/examples.yml

```diff
@@ -41,6 +41,21 @@ buildspecs:
       - "Multi Line description"
       - "is not accepted"
 
+
+  invalid_summary_1:
+    type: script
+    executor: generic.local.bash
+    description: "test"
+    summary: "test"
+
+
+  invalid_summary_2:
+    type: script
+    executor: generic.local.bash
+    description: 1
```

Comment:
For description field I would provide a description of this test. We are not testing description field in this example.

Say for instance "summary field must be a string "